components/tooltip/README.md

```diff
@@ -7,19 +7,32 @@ Tooltips display additional information when users focus or hover on a point of
 <script type="module">
   import '@brightspace-ui/core/components/button/button.js';
   import '@brightspace-ui/core/components/tooltip/tooltip.js';
+  import '@brightspace-ui/core/components/tooltip/tooltip-help.js';
 
   window.addEventListener('load', function () {
     setTimeout(function() {
       var tooltip = document.querySelector('#tooltip');
+      var helpTooltip = document.querySelector('#help-tooltip').shadowRoot.querySelector('d2l-tooltip');
+
       tooltip.showing = true;
+      helpTooltip.showing = true;
     }, 20);
   });
 </script>
 <d2l-button id="tooltip-button">Hover here</d2l-button>
 <d2l-tooltip id="tooltip" for="tooltip-button">
   Tooltip message example
 </d2l-tooltip>
+
+<d2l-tooltip-help id="help-tooltip" text="Helpful label">
+  Contents should elaborate on the label (be short and concise)
+</d2l-tooltip-help>
 ```
+<!-- docs: start hidden content -->
+TODO: Look more into this bug and fix it
+<!-- docs: end hidden content -->
+Note: there is a known defect with tooltips where they are sometimes rendered offset by some amount unexpectedly, the first time that they are shown.
```

Comment:
Can read [here ](https://d2l.slack.com/archives/C03H8KH2FKN/p1656618501840279?thread_ts=1656085775.322529&cid=C03H8KH2FKN)for more context.

Reply:
The defect occurs all the time, not just the first time they're shown (at least for me). So might be good to remove that part.

Reply:
Discussed and clarified this more [here](https://d2l.slack.com/archives/C03H8KH2FKN/p1656947519933479?thread_ts=1656085775.322529&cid=C03H8KH2FKN).